pkgs/development/python-modules/scikit-learn/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
```

Comment:
package already exists!

Reply:
Yep, @jonringer pointed that out. Sorry, I have not gotten around to push my changes yet.